.github/workflows/rust.yaml

```
@@ -0,0 +1,111 @@
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-
+
+      - name: Run tests
+        run: cargo test --verbose
+
+      - uses: actions/cache/save@v4
+        id: cache-test-save
+        if: (success() || failure()) && steps.cache-coverage-restore.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache-coverage-restore" is not defined in object type {cache-test-restore: {conclusion: string; outcome: string; outputs: {cache-hit: string; cache-matched-key: string; cache-primary-key: string}}}

(expression)